app/scripts/modules/amazon/src/loadBalancer/TargetGroup.tsx

```diff
@@ -98,16 +32,29 @@ export class TargetGroup extends React.Component<ITargetGroupProps, ITargetGroup
       />
     ));
 
+    const params = {
+      loadBalancerName: loadBalancer.name,
+      region: targetGroup.region,
+      accountId: targetGroup.account,
+      name: targetGroup.name,
+      vpcId: targetGroup.vpcId,
+      provider: targetGroup.cloudProvider,
+    };
+
     return (
-      <div className="target-group-container container-fluid no-padding" ref={this.refCallback}>
-        <div className={`clickable clickable-row row no-margin-top target-group-header${this.state.active ? ' active' : ''}`} onClick={this.loadDetails}>
-          <div className="col-md-8 target-group-title">
-              {targetGroup.name}
-          </div>
-          <div className="col-md-4 text-right">
-            <HealthCounts container={targetGroup.instanceCounts}/>
-          </div>
-        </div>
+      <div className="target-group-container container-fluid no-padding">
+        <UISrefActive class="active">
+          <UISref to=".targetGroupDetails" params={params}>
```

Comment:
This is how you use `UISref`.  The first dom element rendered (again, the `<div>`) has a click handler added to it.